### DISCUSSION TITLE
Is the RealTime OpenAi voice capabilities in mind to integrate on SK

### DISCUSSION THREAD

**Discussion by nonoc** [microsoft/semantic-kernel#9129] [General]
is that possible to integrate any kind of WebSocket to allow to connect in realtime, any idea, is that even possible in future on SK?

**Comment by rogerbarreto** [comment]
That's correct, we plan on integrating this capabilities as SK Abstractions for Streaming Input/Output communication, currently we don't have any ETAs to provide but it is in our radar even before OpenAI's releasing their RealTime API, which also helps us to architect abstractions that can embrace the current OpenAI's vision for this type of transport.

**Reply by nonoc** [reply]
Well, I was testing today and to be honest it is a bit expensive to implement that technology in any kind of business today. In my opinion, OpenAI will need to decrease or offer a Mini realtime voice... crazy times ins't it? thank you Roger.
